static/app/components/group/suggestedOwners/ownershipRules.tsx

```diff
@@ -3,26 +3,96 @@ import {ClassNames} from '@emotion/react';
 import styled from '@emotion/styled';
 
 import {openCreateOwnershipRule} from 'app/actionCreators/modal';
+import Access from 'app/components/acl/access';
 import GuideAnchor from 'app/components/assistant/guideAnchor';
 import Button from 'app/components/button';
+import FeatureBadge from 'app/components/featureBadge';
 import Hovercard from 'app/components/hovercard';
-import {IconQuestion} from 'app/icons';
+import {Panel} from 'app/components/panels';
+import {IconClose, IconQuestion} from 'app/icons';
 import {t} from 'app/locale';
 import space from 'app/styles/space';
-import {Organization, Project} from 'app/types';
+import {CodeOwner, Organization, Project} from 'app/types';
+import {trackAdvancedAnalyticsEvent} from 'app/utils/advancedAnalytics';
 
 import SidebarSection from '../sidebarSection';
 
 type Props = {
   project: Project;
   organization: Organization;
   issueId: string;
+  codeowners: CodeOwner[];
+  handleCTAClose: () => void;
+  isDismissed: boolean;
 };
 
-const OwnershipRules = ({project, organization, issueId}: Props) => {
+const OwnershipRules = ({
+  project,
+  organization,
+  issueId,
+  codeowners,
+  isDismissed,
+  handleCTAClose,
+}: Props) => {
   const handleOpenCreateOwnershipRule = () => {
     openCreateOwnershipRule({project, organization, issueId});
   };
+  const showCTA =
+    organization.features.includes('integrations-codeowners') &&
+    !codeowners.length &&
+    !isDismissed;
+
+  const createRuleButton = (
+    <Access access={['project:write']}>
+      <GuideAnchor target="owners" position="bottom" offset={space(3)}>
+        <Button onClick={handleOpenCreateOwnershipRule} size="small">
+          {t('Create Ownership Rule')}
+        </Button>
+      </GuideAnchor>
+    </Access>
+  );
+
+  const codeOwnersCTA = (
+    <Container dashedBorder>
+      <HeaderContainer>
+        <Header>{t('Codeowners sync')}</Header> <FeatureBadge type="beta" noTooltip />
+        <DismissIcon size="xs" onClick={() => handleCTAClose()} />
+      </HeaderContainer>
+      <Content>
+        {t(
+          'Import GitHub or GitLab CODEOWNERS files to automatically assign issues to the right people.'
+        )}
+      </Content>
+      <Actions>
+        <SetupButton
+          size="small"
+          priority="primary"
+          href={`/settings/${organization.slug}/projects/${project.slug}/ownership/`}
+          onClick={() =>
+            trackAdvancedAnalyticsEvent('integrations.code_owners_cta_setup_clicked', {
+              project_id: project.id,
+              organization,
+            })
+          }
+        >
+          {t('Set It Up')}
+        </SetupButton>
+        <DocsButton
+          size="small"
+          target="_blank"
```

Comment:
use the https://github.com/getsentry/sentry/blob/b19a2b469e902efdf30b1a1190694d1732b585cb/static/app/components/button.tsx#L29 prop for this and you can drop `target`